### ISSUE TITLE
5-4, form이 array 와 tree를 동시에 구성성분으로 가질 수 있나요?

### ISSUE THREAD

**Issue by haekyu** [snu-sf-class/PP2015f_TA#95]
예를 들어, 
t1이 '('tree W W W W)이고
a1이 '('array (B B) (B B)) 라고 했을 때,

'(tree a1 t1 a1 t1) 처럼 생긴 form이 존재할 수 있나요?
'(tree ((B B)(B B)) (W W W W) ((B B)(B B)) (W W W W)) 같은 것도 존재할 수 있나요?

**Comment by JadeGrande** [comment]
내용 수정합니다!
https://github.com/snu-sf-class/PP2015f_TA/issues/89
에 있는 내용을 보면 그런 것 같네요^^

**Comment by haekyu** [comment]
네, 저도 저 질문내용보고 헷갈렸어요@_@. 질문을 좀 더 자세히 적으면, glue의 input으로는 array와 tree가 함께 존재할 수 있는 것 같은데, form 자체도 array와 tree가 함께 있을 수 있는지 궁금했습니다.

**Comment by kim-yoonseung** [comment]
form 안의 구현은 selfgrader에 적힌 대로 통일되어 있어야 합니다.
array는 본질적으로 내부에 tree를 가질 수 없고, tree의 경우는 가능하긴 하지만 이번 숙제에서는 내부도 모두 tree입니다.

(참고로 #89는 glue 함수의 인풋으로 들어오는 tree들의 구현이 통일되어있지 않을 수 있다는 내용입니다.)
